packages/tldraw/src/components/ToolsPanel/BackToContent.tsx

```diff
@@ -39,7 +41,9 @@ export const BackToContent = React.memo(function BackToContent() {
 
   return (
     <BackToContentContainer id="TD-Tools-Back_to_content" style={{ ...style }}>
-      <RowButton onClick={app.zoomToContent}>Back to content</RowButton>
+      <RowButton onClick={app.zoomToContent}>
+        {intl.formatMessage({ id: 'zoom.to.content' })}
```

Comment:
If it's an html element it's better to use 
<FormattedMesssage id="zoom.to.content" />